libraries/AP_HAL_ChibiOS/hwdef/OrqaF405Pro/README.md

```diff
@@ -0,0 +1,83 @@
+# Orqa FC 3030 F405 Flight Controller
+
+The Orqa FC 3030 F405 is a flight controller produced by [Orqa](https://enterprise.orqafpv.com/).
+
+## Features
+
+ - MCU - STM32F405 32-bit processor
+ - IMU - MPU6000
+ - Barometer - DPS310
+ - OSD - AT7456E
+ - Onboard Flash: 128Mbit
+ - 4x UARTs
+ - 8x PWM Outputs (8 Motor Output)
+ - Battery input voltage: 2S-6S
+ - BEC 5V 2A
+ - BEC 10V 2A
+
+## Pinout
+
+![Orqa FC 3030 F405 Board](OrqaFC3030F405_Board.png "Orqa FC 3030 F405")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (RX/GHST, DMA-enabled)
+ - SERIAL3 -> UART3 (GPS, DMA-enabled)
+ - SERIAL5 -> UART5 (ESC Telemetry)
+ - SERIAL6 -> UART6 (Sik/MAVLink)
+
+## RC Input
+
+RC input is configured for GHST by default on the T1 (UART1_TX) pin. To support other RC protocols set SERIAL1_OPTIONS to 0. This will support all serial RC protocols except SBUS. For PPM and SBUS support on UART1_RX set BRD_ALT_CONFIG to 1.
```

Comment:
dont you need to set half duplex also???

Reply:
It's set by default in the params